Sources/Client/Utils/Atomic.swift

```diff
@@ -33,13 +33,11 @@ public final class Atomic<T> {
     /// Set a value.
     public func set(_ newValue: T?) {
         queue.async(flags: .barrier) { [weak self] in
-            let oldValue = self?.value
-            self?.value = newValue
-            self?.valueChanged(newValue, oldValue)
+            self?.updateValue(from: self?.value, to: newValue)
         }
     }
     
-    /// Get the value.
+    /// Get a value.
```

Comment:
"the value" is actually the correct use :D

Reply:
Anything else? :)